recipe/meta.yaml

```diff
@@ -42,7 +42,7 @@ build:
 
   {% if mpi != 'nompi' %}
   run_exports:
-    - {{ name }} * {{ mpi_prefix }}_{{ tempest }} *
+    - {{ name }} * {{ mpi_prefix }}_{{ tempest }}_*
```

Comment:
@kkiesling, you are right in https://github.com/conda-forge/moab-feedstock/pull/61#issuecomment-1105760333.  There should be an underscore, not a space.  See:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-mpi-variants

I think including ``_{{ tempest }}`` is definitely correct but I think the missing underscore is the root cause of the problem you're seeing.